### PR TITLE
Generate settings.js

### PR DESCRIPTION
**Pull request by lwander**
@danielpeach or @duftler or @jtk54 PTAL

### CODE REVIEW

**Comment by lwander** [comment]
Closes https://github.com/spinnaker/halyard/issues/131

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java [hunk 1]
```diff
@@ -51,6 +51,8 @@
 
   Features features = new Features();
 
+  String timezone = "America/Los_Angeles";
```
Comment (danielpeach) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java]: This should probably be configurable eventually.

File: halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/DeckProfile.java [hunk 2]
```diff
@@ -44,4 +50,42 @@ public String getProfileFileName() {
...
+    // Configure GCE
+    GoogleProvider googleProvider = deploymentConfiguration.getProviders().getGoogle();
+    bindings.put("google.default.account", googleProvider.getPrimaryAccount());
+    bindings.put("google.default.region", "us-central1");
+    bindings.put("google.default.zone", "us-central1-f");
+
+    // Configure Appengine
+    AppengineProvider appengineProvider = deploymentConfiguration.getProviders().getAppengine();
+    bindings.put("appengine.default.account", appengineProvider.getPrimaryAccount());
+    bindings.put("appengine.enabled", Boolean.toString(appengineProvider.getPrimaryAccount() != null));
```
Comment (danielpeach) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/DeckProfile.java]: Does this write out `'true'` or `'false'` as strings to setting.js?

Comment (lwander) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/DeckProfile.java]: No, it writes them as `true` or `false` without the quotes.

**Pull request merged**